dev.dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM node:16-alpine
+FROM 169942020521.dkr.ecr.eu-west-2.amazonaws.com/base/node-16:alpine-builder
+FROM 169942020521.dkr.ecr.eu-west-2.amazonaws.com/base/node-16:alpine-runtime
 
-WORKDIR /app
-
-EXPOSE 3000
\ No newline at end of file
+COPY api-enumerations ./api-enumerations
+RUN cp -r ./dist/* ./ && rm -rf ./dist
+CMD ["--inspect=0.0.0.0:9229", "server.js"]
```

Comment:
This is the debugging port needs, should be updated to the development port

Reply:
This dockerfile is for local development only which is why I've put this debug access here. The actual port for use is still in the line below.
When rea-web is used as a standard service in docker, the other Dockerfile is used without the debig port